### PR TITLE
F/pydantic consumer

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

Update investigator to be compatible with latest version of messaging.

### CODE REVIEW

**Comment by KPostOffice** [comment]
Closes: #421

File: consumer.py [hunk 1]
```diff
@@ -56,27 +56,26 @@
...
-run_module("thoth.investigator.package_extract_trigger.investigate_package_extract_trigger")
-run_module("thoth.investigator.package_released.investigate_package_released")
-run_module("thoth.investigator.provenance_checker_trigger.investigate_provenance_checker_trigger")
-run_module("thoth.investigator.qebhwt_trigger.investigate_qebhwt_trigger")
-run_module("thoth.investigator.si_unanalyzed_package.investigate_si_unanalyzed_package")
-run_module("thoth.investigator.solved_package.investigate_solved_package")
-run_module("thoth.investigator.unresolved_package.investigate_unresolved_package")
-run_module("thoth.investigator.unrevsolved_package.investigate_unrevsolved_package")
-run_module("thoth.investigator.update_provide_source_distro.investigate_update_provide_source_distro")
+run_module("thoth.investigator.advise_justification.__init__")
```
Comment (harshad16) [inline consumer.py:60]: all lgtm,
just unsure of this.
maybe we should just do a run of this in test and see.

Comment (KPostOffice) [reply consumer.py:60]: sure, will /deploy do the trick. I'm not sure exactly how that works since there are two deployments dependent on this repository.

Comment (harshad16) [reply consumer.py:60]: what are the two deployments' dependencies? and yes that should get the image built for investigator and get it deployed in the test cluster.

Comment (KPostOffice) [reply consumer.py:60]: Hey sorry. It's `investigator` and `investigator_metrics`

Comment (KPostOffice) [reply consumer.py:60]: @harshad16 The above does run all the modules. I tested locally by just adding a print statement to one of the modules

Comment (harshad16) [reply consumer.py:60]: excellent, thanks @KPostOffice

**Review by harshad16 — APPROVED** [review]
/lgtm

**Pull request merged**